doc/whats_new.rst

```diff
@@ -184,6 +184,8 @@ Enhancements
    - Added optional parameter ``warm_start`` in
      :class:`linear_model.LogisticRegression`. If set to True, the solvers ``lbfgs``, ``newton-cg`` and ``sag`` will be initialized with the coefficients computed in the previous fit. By `Tom Dupre la Tour`_.
 
+   - Added ``laplacian kernel`` in :class:`metrics.pairwise`.  By `Clyde Fare <https://github.com/Clyde-fare>`_.
```

Comment:
I don't think this is correct. I think you want `   - Added :func:metrics.laplacian_kernel`. By `Clyde \`Clyde Fare <https://github.com/Clyde-fare>\`_.`